### PR TITLE
Update Python CLI Installation

### LINKED ISSUE #1129: Update Python CLI Standalone info
The python cli standalone exe for Windows is no longer being released. Documentation needs to be updated to remove this reference.

### PR DESCRIPTION
**Pull request by rcarteraz**
### What
- Updated the installation instructions for the Meshtastic Python CLI.
- Reorganized the content with appropriate headings for better structure.
- Removed reference to the Windows standalone executable, which is no longer available.

### Why
- Provide clearer and more up-to-date instructions for users to install the Meshtastic Python CLI.
- Improve the overall readability and navigation of the documentation by using headings and organizing the content logically.
- Reflect the accurate availability of standalone executables, as the Windows version is no longer provided. This closes #1129.

### CODE REVIEW

**Review by pdxlocations — APPROVED** [review]

File: docs/software/python-cli/installation.mdx [hunk 1]
```diff
@@ -4,35 +4,40 @@ title: Meshtastic Python CLI installation
 sidebar_label: Installation
 slug: /software/python/cli/installation
 sidebar_position: 1
+description: This page offers comprehensive instructions on methods of installing the Meshtastic Python CLI across different operating systems.
```
Comment (tekstrand) [inline docs/software/python-cli/installation.mdx:7]: 👏

File: docs/software/python-cli/installation.mdx [hunk 2]
```diff
@@ -4,35 +4,40 @@ title: Meshtastic Python CLI installation
...
-Make sure that the `PATH variable` also gets installed by checking the box while installing python. If you don't, python may not be available and you may not be able to call `meshtastic` from your CLI. If you do forget to check that box, you will need to install the path environment variable for python on your operating system.
-:::
-:::important
-You may need to install a driver from Silicon Labs for the [CP210X USB to UART bridge](https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers)
+You can install the Meshtastic CLI using one of the following methods:
+
+1. **Python Package Installer (pip)**: Installation can be easily done through the [Python package installer pip](https://pypi.org/project/meshtastic). Make sure to use pip version 20 or later by running `pip install --upgrade pip` if necessary.
+2. **Standalone Executable (Ubuntu only)**: A single executable file for Ubuntu is available on the [Releases](https://github.com/meshtastic/Meshtastic-python/releases) page. See [Standalone](#standalone-installation-ubuntu-only) below for instructions.
+
+### Prerequisites
```
Comment (tekstrand) [inline docs/software/python-cli/installation.mdx:26]: Do we typically put the prereqs at the end in our docs? Just wondering if it makes more sense to put them first so people don't proceed if they don't have these complete.

**Review by garthvh — APPROVED** [review]
Looks good

File: docs/software/python-cli/installation.mdx (hunk 2 above)
Comment (tekstrand) [reply docs/software/python-cli/installation.mdx:26]: Ah I see now! Yes a misunderstanding on my part.

I don't think either approach is wrong or right. If it's duplicate information, we just have to be sure it's useful to have it twice, IMO. You have more context/experience so I think you should make the call between those two.

**Review by tekstrand — APPROVED** [review]
I like this format, nice work

**Pull request merged**